lib/Util/Notices/InactiveSubscribersNotice.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace MailPoet\Util\Notices;
+
+use MailPoet\Models\Subscriber;
+use MailPoet\Settings\SettingsController;
+use MailPoet\Util\Helpers;
+use MailPoet\WP\Functions as WPFunctions;
+use MailPoet\WP\Notice;
+
+class InactiveSubscribersNotice {
```

Comment:
No tests? Why do you think we don't need them for this class?

Reply:
The ticket says no tests needed but thinking about it... I'd add some integration tests at least.